### DISCUSSION TITLE
It's time to add an Export feature for the Android platform

### DISCUSSION THREAD

**Discussion by bitesgame** [godotengine/godot-proposals#8782] [Platforms]
### Describe the project you are working on

I'm making a 2D game using Android Godot

### Describe the problem or limitation you are having in your project

on the android platform godot editor the export button is accidentally closed

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

![Screenshot_2024-01-03-15-54-43-783_com android chrome](https://github.com/godotengine/godot-proposals/assets/65835193/7c50a7f6-2a34-4f9d-a7bc-e65187b1aa3f)

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

T/A

### If this enhancement will not be used often, can it be worked around with a few lines of script?

T/A

### Is there a reason why this should be core and not an add-on in the asset library?

AndroidIDE has helped me in creating Android applications on Android itself
```[tasklist]
### Tasks
```

**Comment by AThousandShips** [comment]
See:
* https://github.com/godotengine/godot/issues/73130

This isn't a proper proposal, you have to fill in all the questions, so converting to a discussion instead

**Reply by bitesgame** [reply]
I'm sorry because my English skills are quite bad, so I don't really understand the instructions, so forgive me(¡-¡)

**Reply by AThousandShips** [reply]
That's okay, please use any translation help you can find 🙂

**Reply by bitesgame** [reply]
I have found an IDE tool for the Android platform, namely AndroidIDE, so what do you think, because I don't have a desktop or laptop to compile the project, so when I want to export the Godot project, I'm confused when I want to compile it :)

**Reply by AThousandShips** [reply]
See the other issue, it talks about the limitations 🙂, if it's possible to compile for android on android this could be an option, but it depends on how it works and if it's compatible

**Comment by bitesgame** [comment]
![Screenshot_2024-01-03-21-37-55-984_com android chrome](https://github.com/godotengine/godot-proposals/assets/65835193/75b8fa50-4ded-4c04-8458-53e057ed04ca)

they use JDK 11 and JDK 17 and if I'm not mistaken the Android SDK used is 34.0.1 and also only supports AGP 7.2.0 to newer versions, and as for whether it's compatible or not, I'm asking the rest of the community to what extent it supports and is compatible. I don't know yet because it's still under active development

**Reply by AThousandShips** [reply]
This is under the GPLv3 so it's not compatible with Godot on itself, and any games made with it *might* be also forced to be under that license, depends I haven't read the details, but if that's the case I don't think we can or should support this officially

Also please link to this page so people don't have to search

**Reply by bitesgame** [reply]
oh, good and thank you that's really helpful, because I'm an amateur so I sometimes forget the different terms in licensing for software 😅

**Reply by AThousandShips** [reply]
Could you please add the link 🙂

**Reply by bitesgame** [reply]
Sorry, what you asked for was a link to the Android IDE? if yes then here is the link:
 https://github.com/AndroidIDEOfficial/AndroidIDE

and forgive me again because my English skills are bad, using translate is not very useful, there are some sentences or words that cannot be translated

**Comment by syntaxerror247** [comment]
Exporting from Android Editor is now supported. See https://github.com/godotengine/godot/pull/93526 for more details.
